haxball.js

```diff
@@ -22,22 +27,52 @@ function updateAdmins() {
 room.onPlayerJoin = function(player) {
   room.sendAnnouncement( "Elo " + player.name);
   updateAdmins();
+  
+  if(isBambik(player.name)){bambiksInGame.push(player)} else {playersInGame.push(player);}
+  
+  var scores = room.getScores();
+  if(scores == null && (bambiksInGame.length + playersInGame.length) == 6){handleTeamSelecting(shuffle(bambiksInGame), shuffle(playersInGame));}

```

Comment:
What if (bambiksInGame.length + playersInGame.length) >= 8, aka huge?

Reply:
huge stadium handling will be implemented in next sprint